package.json

```
@@ -76,7 +76,7 @@
         "@typescript-eslint/parser": "^5.9.0",
         "babel-jest": "^27.4.6",
         "babel-loader": "^8.2.3",
-        "chalk": "^4.1.2",
+        "chalk": "^5.0.0",
```

Let's kee chalk at v4 since their v5 is for mjs
```suggestion
        "chalk": "^4.1.2",
```

---

yeah I saw they change to ESM.

---

It isn't the only problem. The storybook update introduces new changes and deprecates some keys on the docs.container. I need to check how to solve that issue.